features/Makefile

```diff
@@ -1,25 +1,37 @@
 
-.PHONY: test, ginkgo
+export FEATURES?=mcp performance sctp
 
-export FEATURES?=sctp 
-
-deps:
+.PHONY: deps-update
+deps-update:
 	go mod tidy && \
 	go mod vendor
 
 ginkgo:
 	go get github.com/onsi/ginkgo/ginkgo
 
+.PHONY: functests
 functests: ginkgo
 	GOFLAGS=-mod=vendor ginkgo functests -- -junit /tmp/artifacts/unit_report.xml
 
+.PHONY: unittests
 unittests:
 	GO_PACKAGES=$$(go list ./...| grep -v /functests); \
 	if [ -n "$$GO_PACKAGES" ]; then go test $$GO_PACKAGES; fi; 
 
-deploy:
-	for f in $(FEATURES) ; \
-    do \
-	  echo "deploying $$f"; \
-      ./$$f/deploy.sh; \
-    done \
+.PHONY: deploy
+deploy: generate
+	for f in $(FEATURES); do \
+		if [ -f ./$$f/deploy.sh ]; then \
```

Comment:
Neat!

Reply:
Case sensitive though.. and I would add else with echo "WARN: No such feature $f" or something

Reply:
will add the WARN message

Reply:
done